src/Components/DisplayMovie.js

```diff
@@ -0,0 +1,31 @@
+const DisplayMovies = (props) => {
+
+    return (
+
+        <div className="MovieResult">
+            {
+                props.movie === undefined ?
+
```

Comment:
consider re-formatting ternary statement to code easier to read. maybe adjust the indentation.